src/components/Header.js

```diff
@@ -118,7 +118,11 @@ export const HeaderTemplate = ({ navItems = [] }) => {
                 </a>
               </li>
               <li>
-                <a href="#" className="primary" rel="noopener noreferrer">
+                <a
+                  href="#"
+                  className="btn btn-primary nav-item"
+                  rel="noopener noreferrer"
+                >
```

Comment:
@maymax777 , this line is also duplicated with PR [#356](https://github.com/keep-network/website/pull/356/files) .